main.cpp

```diff
@@ -36,6 +37,7 @@ int main(int argc, char *argv[])
     parser.addHelpOption();
     parser.addVersionOption();
     parser.addPositionalArgument("media", QCoreApplication::translate("KokoVP", "Media files to play"));
+    parser.addPositionalArgument("urls", QCoreApplication::translate("KokoVP", "URLs to play"));
```

Comment:
How multiple position arguments lists should work at all? Maybe just accept media by local file paths and by urls?